SWIG/null.i

```diff
@@ -60,11 +60,13 @@ double nullDouble() { return Null<double>(); }
         $1 = Null<double>();
     else if (PyFloat_Check($input))
         $1 = PyFloat_AsDouble($input);
+    else if (PyLong_Check($input))
+        $1 = int(PyLong_AsLong($input));
```

Comment:
should cast to double, not int. The latter will truncate some values.
```suggestion
        $1 = double(PyLong_AsLong($input));
```

Reply:
Doubles will match the clause above before getting here, but you're right, of course.

Reply:
Yes, specifically it would truncate long values from PyLong_AsLong that do not fit into ints :)